bin/ci

```diff
@@ -148,6 +148,9 @@ with_build_env() {
     x86_64)
       export DOCKER_TEST_IMAGE="$DOCKER_TEST_PREFIX-build"
       ;;
+    x86_64-musl)
```

Comment:
Shouldn't it be the other way around to mimic the target triple?

```suggestion
    musl-x86_64)
```

Reply:
How does that mimic? The target triples are `x86_64-linux-gnu` and `x86_64-linux-musl`.

Reply:
I got confused with the name of the .tar.gz which are *-linux-x86_64.